content/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists.md

```diff
@@ -80,4 +80,3 @@ Alternatively, you can drag and drop a text file from your desktop directly into
 ![Drop-down menu to select gist visibility]{% ifversion ghae %}(/assets/images/help/gist/gist-visibility-drop-down-ae.png){% else %}(/assets/images/help/gist/gist-visibility-drop-down.png){% endif %}
```

Comment:
I don't think the `gist_name_description` and `gist_test_box` images are necessary. When you navigate to the Gist homepage, there was only 3 fields. Folks should be able to find them by description alone. 

Instead, let's remove both those images and update the procedural steps to clarify which fields we're talking about. I can't leave this comment on the appropriate lines unfortunately 😅 but something like this:

1. In the "Gist description" field, type a description for your gist.
2. In the "Filename including extension" field,  type a file name for your gist, including the file extensions.
3. In the file contents field, type the text of your gist.